src/entity/user_docs.entity.ts

```
@@ -15,40 +15,56 @@ const encryptionTransformer = new EncryptionTransformer(configService);
-  doc_data: Record<string, unknown> | null;
+	@Column({ type: 'varchar', length: 1500, nullable: true })
+	doc_data_link: string;
 
-  @Column({ type: 'varchar', length: 100 })
-  doc_datatype: string;
+	@Column({
+		type: 'text',
+		nullable: true,
+		transformer: encryptionTransformer,
+	})
+	doc_data: Record<string, unknown> | null;
```

ESLint

[error] Expected indentation of 2 tabs but found 1.

(indent)